pkg/controller/controller.go

```diff
@@ -973,46 +973,46 @@ func (s *Service) ControllerGetVolume(ctx context.Context, req *csi.ControllerGe
 		// check if filesystem exists
 		fs, err := s.Arrays()[arrayID].Client.GetFS(ctx, id)
 		if err != nil {
-			return nil, status.Errorf(codes.NotFound, "failed to find filesystem: %s with error: %v", id, err.Error())
-		}
-
-		// get exports for filesystem if exists
-		nfsExport, err := s.Arrays()[arrayID].Client.GetNFSExportByFileSystemID(ctx, fs.ID)
-		if err != nil {
-			if apiError, ok := err.(gopowerstore.APIError); !ok || !apiError.NotFound() {
-				return nil, status.Errorf(codes.NotFound, "failed to find nfs export for filesystem with error: %v", err.Error())
-			}
+			abnormal = true
```

Comment:
Return abnormal only for 404 - FS not found. Return error in case of any other status code

Reply:
done